plugin/storage/es/spanstore/writer.go

```diff
@@ -87,12 +104,16 @@ func spanIndexName(span *model.Span) string {
 // Check if index exists, and create index if it does not.
 func (s *SpanWriter) checkAndCreateIndex(indexName string, jsonSpan *jModel.Span) error {
 	// TODO: We don't need to check every write. Try to pull this out of WriteSpan.
+	start := time.Now()
 	exists, err := s.client.IndexExists(indexName).Do(s.ctx)
+	s.writerMetrics.exists.Emit(err, time.Since(start))
```

Comment:
Not a fan of how this looks... unfortunately, I'd prefer it if we have a separate metrics function for each of the elasticSearch client services. (ie a Do() wrapper for each service that handles the metrics and is a callthrough for the real Do function.

:D Maybe TODO it for now and tackle it after